### PR TITLE
GN-4261: addition of an address variable type

### PR DESCRIPTION
**Pull request by elpoelma**
### Overview
This PR introduces an implementation of an address variable. It includes an address node-spec and node-view, as well as address insert and edit components. Using the address-edit component, you search for an address in the ember-power-select dropdown. The edit component searches for addresses through the location register (`https://geo.api.vlaanderen.be/geolocation/v4/Location`) and then later resolves the addresses through the address-register (`https://basisregisters.vlaanderen.be/api/v1/adressen`).
When searching for an address, the user can select whether they want to include a housenumber or not.
The outputted RDFa of an address is based on the https://data.vlaanderen.be/ns/adres/ namespace.

Additionally, this PR also removes the old address plugin.

##### connected issues and PRs:
Jira: https://binnenland.atlassian.net/browse/GN-4261?atlOrigin=eyJpIjoiNjIxOGI4ZTM4MDk0NDI0MGFmMjQyMmQwMDNmODE0NjEiLCJwIjoiaiJ9

### How to test/reproduce
- Insert an address variable using the side menu
- Click on the address variable and check if the edit-card appears
- Search for an address in the edit card and insert it
- Reload to check if the address is correctly serialized and parsed by prosemirror
- Click on the address again to edit it. Check if the dropdown is prefilled by the address inserted.
- Toggle the `include housenumber` and check if the selected address in the dropdown is cleared.
- Search for an address without a housenumber and insert it.
- Note that addresses without a housenumber do not have an `addressRegisterId` attribute, as they are not really an address.

- Check translations
### Challenges/uncertainties
- The styling of the address dropdown still seems a bit weird.

### Checks PR readiness
- [x] UI: works on smaller screen sizes
- [x] UI: feedback for any loading/error states
- [x] Check if dummy app is correctly updated
- [x] Check cancel/go-back flows
- [x] changelog
- [x] npm lint

### CODE REVIEW

File: addon/plugins/variable-plugin/variables/address.ts [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+  static resolve(address: Address, addressRegisterId: string) {
+    return new ResolvedAddress({
+      addressRegisterId,
+      ...address,
+    });
+  }
+}
+
+const constructAddressNode = (address?: Address | ResolvedAddress) => {
```
Comment (abeforgit) [inline addon/plugins/variable-plugin/variables/address.ts:91]: Any chance we can structure this a bit cleaner? 
output looks like this:
image

Would be nice if we could wrap the constituent data in the spans, so we avoid excessive use of the `content` attribute

**Review by abeforgit — APPROVED** [review]
couldn´t really find any problems, just a note about the rdfa structure

File: addon/plugins/variable-plugin/variables/address.ts (hunk 1 above)
Comment (elpoelma) [reply addon/plugins/variable-plugin/variables/address.ts:91]: Yeah that would be cleaner indeed, I'll update it.

Comment (elpoelma) [reply addon/plugins/variable-plugin/variables/address.ts:91]: The data is now wrapped inside the spans, instead of using the content attribute.

**Comment by elpoelma** [comment]
I will convert this PR to a draft, and implement the new design referenced in https://binnenland.atlassian.net/browse/GN-4263?atlOrigin=eyJpIjoiMWJjNWM2ZjEyMTFiNGEyNWI0MzZjYjgwZDQ1ZmZmYjMiLCJwIjoiaiJ9

**Comment by elpoelma** [comment]
> I will convert this PR to a draft, and implement the new design referenced in https://binnenland.atlassian.net/browse/GN-4263?atlOrigin=eyJpIjoiMWJjNWM2ZjEyMTFiNGEyNWI0MzZjYjgwZDQ1ZmZmYjMiLCJwIjoiaiJ9

Update: I have decided to keep this PR as it is, and work on the redesign in a seperate PR and branch. The code and functionality on this branch works, so we might as well merge it when ready.

**Pull request merged**